uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/device/settings/GoogleSettings.java

```diff
@@ -210,4 +202,51 @@ public void activateAdmin() {
             Assert.fail(e.getMessage());
         }
     }
+
+    /**
+     * @return returns the button of date in the settings screen.
```

Comment:
Either provide complete javaodoc.....or just remove it as this is a private method